lightning/src/ln/functional_tests.rs

```diff
@@ -2526,8 +2496,7 @@ fn test_justice_tx() {
 	claim_payment(&nodes[0], &vec!(&nodes[1])[..], payment_preimage_3, 3_000_000);
 
 	{
-		let mut header = BlockHeader { version: 0x20000000, prev_blockhash: Default::default(), merkle_root: Default::default(), time: 42, bits: 42, nonce: 42 };
-		connect_block(&nodes[1], &Block { header, txdata: vec![revoked_local_txn[0].clone()] }, 1);
+		confirm_transaction_at(&nodes[1], &revoked_local_txn[0], nodes[1].best_block_info().1 + 1);
```

Comment:
Would it be possible to add another helper to use when needing a height that is offset from the best block to avoid explicitly calling `best_block_info()`?